safe/engine/test/test_engine.py

```diff
@@ -414,6 +418,45 @@ def test_polygon_hazard_with_holes_and_raster_exposure(self):
 
     test_polygon_hazard_with_holes_and_raster_exposure.slow = True
 
+    def test_user_directory_when_saving(self):
+        # These imports must be inside the test.
+        from PyQt4.QtCore import QCoreApplication, QSettings
+        from qgis.core import QgsApplication
+
+        # noinspection PyCallByClass,PyArgumentList
+        QCoreApplication.setOrganizationName('QGIS')
+        # noinspection PyCallByClass,PyArgumentList
+        QCoreApplication.setOrganizationDomain('qgis.org')
+        # noinspection PyCallByClass
+        QCoreApplication.setApplicationName('QGIS2InaSAFETesting')
```

Comment:
Hi @Gustry, we already have this in get_qgis_app, so you should not specify this for QSetting anymore and just call get_qgis_app in the beginning of the module. This is for an example https://github.com/AIFDR/inasafe/blob/develop/safe/gui/tools/batch/test/test_batch_dialog.py#L27